docs/plugins/build-your-own.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Building Your Own Plugin
+label: Build Your Own
+order: 11
+desc: Starting to build your own plugin? Find everything you need and learn best practices with the Payload plugin template.
+keywords: plugins, template, config, configuration, extensions, custom, documentation, Content Management System, cms, headless, javascript, node, react, express
+---
+
+Building your own plugin is easy, and if you&apos;re already familiar with Payload then you&apos;ll have everything you need to get started.
+
+<Banner type="success">
+  You can either start from scratch or use the [Payload plugin template.](https://github.com/payloadcms/payload-plugin-template)
```

Comment:
Do you need to update this to mention `npx create-payload-app`?

Reply:
Sure do - added now!